pandas/tools/util.py

```diff
@@ -48,3 +48,66 @@ def compose(*funcs):
     """Compose 2 or more callables"""
     assert len(funcs) > 1, 'At least 2 callables must be passed to compose'
     return reduce(_compose2, funcs)
+
+
+def to_numeric(arg, errors='raise', box=True, coerce=None):
+    """
+    Convert argument to a numeric type.
+
+    Parameters
+    ----------
+    arg : string, array of strings, mixed object array
+    errors : {'ignore', 'raise', 'coerce'}, default 'raise'
+        - If 'raise', then invalid parsing will raise an exception
+        - If 'coerce', then invalid parsing will be set as NaT
+        - If 'ignore', then invalid parsing will return the input
+    box : boolean, default True
+        - If True returns a Series
+        - If False returns ndarray of values.
```

Comment:
The `box` keyword here has another meaning as the `box` keyword in `to_datetime`. There, it is to determine of an index or an array is returned (a Series is always returned as a Series, regardless of the value of `box`).

But, the meaning of `to_datetime` is not very useful here I think. @bashtage Do you think this keyword is needed? (because, one option to not have this inconsistency, is maybe to just leave it out here? ) If you want the values itself as the return value, you can always pass it the values yourself instead of passing an argument (`pd.to_numeric(s.values)` instead of `pd.to_numeric(s, box=False)`